README.rst

```diff
@@ -67,44 +66,77 @@ information::
     python manage.py help loadtestdata
 
 
-Using autofixtures as tool for unittests
+Using autofixtures as a tool for unittests
 ========================================
 
-It has proofed that autofixtures have a great use for unittests. It has always
-bugged me that creating complex models for testing their behaviour was
-complicated. Sometimes models have strict restrictions or many related objects
-which they depend on. One solution would be to use traditional fixtures
-dumped from your production database. But while in development when database
-schemes are changing frequently, its hard to maintain all fixtures and to know
-exactly which objects are contained in the dumps etc...
+Testing the behavior of complex models has always bugged me. Sometimes models 
+have many restrictions or many related objects which they depend on. One solution 
+would be to use traditional fixtures dumped from your production database. But 
+while in development when database schemes are changing frequently, it can be time
+consuming and sometimes difficult to deep track of changes and what each dump contains.
 
-Autofixtures to the rescue! It lets you automatically generate models and all
-of their dependencies on the fly. Have a look at the following examples.
+Autofixtures to the rescue! 
 
-Lets start with the very basics. We create an ``AutoFixture`` instance for the
+Let's start with the basics. We create an ``AutoFixture`` instance for the
 ``Entry`` model and tell it to create ten model instances::
 
+(in Python shell)
```

Comment:
You could use the `>>>` notation to indicate that you present an example in the python shell:

```
>>> from autofixture import AutoFixture
>>> fixture = AutoFixture(Entry)
>>> entries = fixture.create(10)
```